[cadence][g3] enable fp32 permute

zonglinpeng opened the pull request:
Summary: titled

Differential Revision: D70796738

mcremon-meta commented on backends/cadence/fusion_g3/operators/op_permute_copy.cpp:
```diff
@@ -156,4 +157,4 @@ Tensor& permute_copy_out(
 } // namespace impl
-} // namespace cadence
\ No newline at end of file
+} // namespace cadence
```
if linter is ok, probably doesn't matter :)

Commit:
link m3 ops and add testcases

Differential Revision: D71155209

zonglinpeng committed:
enable fp32 permute (#9053)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/9053

titled

Reviewed By: mcremon-meta

Differential Revision: D70796738
